From 6568614237d251b9fa72dd082b10974b71c5c38b Mon Sep 17 00:00:00 2001
From: "ola.lilja" <ola.lilja@intel.com>
Date: Tue, 10 Feb 2015 11:31:48 +0100
Subject: [PATCH] Audio: Remove the last part (hdmi-audio) from device_libs

Remove the file platform_byt_audio.c as it does nothing
after removing creation of hdmi-audio platform-device.

Issue: GMINL-3671

Change-Id: I4443216188a0453163111379ed3f0e5ea03d2866
Signed-off-by: ola.lilja <ola.lilja@intel.com>
---
 arch/x86/platform/intel-mid/device_libs/Makefile   |  1 -
 .../intel-mid/device_libs/platform_byt_audio.c     | 28 ----------------------
 2 files changed, 29 deletions(-)
 delete mode 100644 arch/x86/platform/intel-mid/device_libs/platform_byt_audio.c

diff --git a/arch/x86/platform/intel-mid/device_libs/Makefile b/arch/x86/platform/intel-mid/device_libs/Makefile
index 61b64d9..b05c054 100644
--- a/arch/x86/platform/intel-mid/device_libs/Makefile
+++ b/arch/x86/platform/intel-mid/device_libs/Makefile
@@ -1,6 +1,5 @@
 # IPC Devices
 #obj-y += platform_ipc.o
-obj-$(subst m,y,$(CONFIG_SND_SST_PLATFORM)) += platform_byt_audio.o
 obj-$(subst m,y,$(CONFIG_MFD_INTEL_MSIC)) += platform_msic.o
 obj-$(subst m,y,$(CONFIG_SND_MFLD_MACHINE)) += platform_msic_audio.o
 obj-$(subst m,y,$(CONFIG_GPIO_MSIC)) += platform_msic_gpio.o
diff --git a/arch/x86/platform/intel-mid/device_libs/platform_byt_audio.c b/arch/x86/platform/intel-mid/device_libs/platform_byt_audio.c
deleted file mode 100644
index 36ce2ee..0000000
--- a/arch/x86/platform/intel-mid/device_libs/platform_byt_audio.c
+++ /dev/null
@@ -1,28 +0,0 @@
-/*
- * platform_byt_audio.c: Baytrail audio platform data initilization file
- *
- * (C) Copyright 2013 Intel Corporation
- * Author: Omair Md Abudllah <omair.m.abdullah@intel.com>
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; version 2
- * of the License.
- */
-
-#include <linux/kernel.h>
-#include <linux/init.h>
-#include <linux/platform_device.h>
-
-
-static int __init byt_audio_platform_init(void)
-{
-	struct platform_device *pdev;
-
-	pr_debug("%s: Enter.\n", __func__);
-
-	pdev = platform_device_register_simple("hdmi-audio", -1, NULL, 0);
-
-	return 0;
-}
-device_initcall(byt_audio_platform_init);
-- 
1.9.1

